app/frontend/src/cljs/teet/ui/table.cljs

```diff
@@ -88,37 +88,41 @@
 
         show-more! #(swap! show-count + 20)]
     (r/create-class
-     {:component-will-receive-props (fn [_this _new-props]
-                                      ;; When props change, reset show count
-                                      ;; filters/results have changed
-                                      (reset! show-count 20))
-      :reagent-render
-      (fn [{:keys [on-row-click filters data columns filter-type get-column format-column key]
-            :or {filter-type {}
-                 get-column get}}]
-        (let [[sort-col sort-dir] @sort-column]
-          [:<>
-           [Table {}
-            [listing-header {:sort! sort!
-                             :sort-column sort-col
-                             :sort-direction sort-dir
-                             :filters filters
-                             :columns columns
-                             :filter-type filter-type}]
-            [TableBody {}
-             (doall
-              (for [row (take @show-count
-                              ((if (= sort-dir :asc) identity reverse)
-                               (sort-by #(get-column % sort-col) data)))]
-                ^{:key (get row key)}
-                [TableRow {:on-click #(on-row-click row)
-                           :class (<class row-style)}
-                 (doall
-                  (for [column columns]
-                    ^{:key (name column)}
-                    [TableCell {}
-                     (format-column column (get-column row column) row)]))]))]]
-           [scroll-sensor/scroll-sensor show-more!]]))})))
+      {:component-will-receive-props (fn [_this _new-props]
+                                       ;; When props change, reset show count
+                                       ;; filters/results have changed
+                                       (reset! show-count 20))
+       :reagent-render
+       (fn [{:keys [on-row-click filters data columns filter-type get-column get-column-compare format-column key]
+             :or {filter-type {}
+                  get-column get}}]
+         (let [[sort-col sort-dir] @sort-column]
+           [:<>
+            [Table {}
+             [listing-header {:sort! sort!
+                              :sort-column sort-col
+                              :sort-direction sort-dir
+                              :filters filters
+                              :columns columns
+                              :filter-type filter-type}]
+             [TableBody {}
+              (doall
+                (for [row (take @show-count
+                            ((if (= sort-dir :asc) identity reverse)
+                             (if get-column-compare
+                               (sort-by #(get-column % sort-col)
+                                 (fn [x y]
```

Comment:
Doesn't the `get-column-compare` take one argument (a column) and return a comparison function?
apply is calling it here with 2 values (x and y)

Reply:
Right, fixed.